lib/pubid/iso/identifier/international_workshop_agreement.rb

```diff
@@ -0,0 +1,28 @@
+module Pubid::Iso
+  module Identifier
+    class InternationalWorkshopAgreement < Base
+      def_delegators 'Pubid::Iso::Identifier::InternationalWorkshopAgreement', :type
+
+      TYPED_STAGES = {
+        dis: {
+          abbr: "DIWA",
+          name: "Draft International Workshop Agreement",
+          harmonized_stages: %w[40.00 40.20 40.60 40.92 40.93],
+        },
+        fdis: {
```

Comment:
There is no "Final Draft" stage for IWA. This is described in the Typed Stages Excel file:
<img width="1018" alt="Screen Shot 2022-12-14 at 10 27 29 PM" src="https://user-images.githubusercontent.com/11865/207607580-cbfe6067-7ac2-482e-acef-5cc206a3b7ec.png">

Can you please refer to the file?

Reply:
I'll add this change, but excel file version I have looks different:
<img width="1177" alt="image" src="https://user-images.githubusercontent.com/5849/207828644-4dd8a110-92f5-40e1-ac96-97ce16befad9.png">

Do you have other updates?

Reply:
I’ve previously uploaded the latest version to a ticket. I’ll have to find that again.

Reply:
It is now at #176 .